plugins/modules/nd_pcv.py

```diff
@@ -43,7 +43,7 @@
     aliases: [ site ]
   file:
     description:
-    - Optional parameter if creating new pre-change analysis from file.
+    - Optional parameter if creating new pre-change analysis from file. XML and JSON files are supported. If no file extension is provided, the file is assumed to be JSON.
```

Comment:
```suggestion
    - Optional parameter if creating new pre-change analysis from file.
    - XML and JSON files are supported. If no file extension is provided, the file is assumed to be JSON.
```
To fix the long line sanity issue.